src/main/resources/application.properties

```diff
@@ -1 +1,10 @@
-
+spring.datasource.driver-class-name=com.mysql.jdbc.Driver
+spring.datasource.url = jdbc:mysql://127.0.0.1:3306/manager_shop
+spring.datasource.username=root
+spring.datasource.password=123456
+# Hibernate Properties
+# The SQL dialect makes Hibernate generate better SQL for the chosen database
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL5InnoDBDialect
+# Hibernate ddl auto (create, create-drop, validate, update)
+spring.jpa.hibernate.ddl-auto = update
+logging.level.org.hibernate = ERROR
```

Comment:
comment xuống dây giải thích cho cậu 2 dòng: 9 và 10 là để làm gì nhé.

Reply:
dòng 9: cấu hình update là để tự động tạo và cậu nhật bảng trong sql khi chạy ứng dụng
dòng 10: là cấu hình lever hiển thị lỗi. ở đây là error: chỉ in ra thông báo lỗi mà không có debug và các thông báo khác 
để giảm dung lượng và tăng tốc của log

Reply:
ok. cần hiểu log là gì nhá. chính là cái system.out.println đấy. có 1 số loại log cũng cần biết qua: log info, log error, log debug...